cou/utils/app_utils.py

```diff
@@ -50,6 +50,31 @@ async def upgrade_packages(
         await model.run_on_unit(unit_name=unit, command=command, timeout=600)
 
 
+async def get_instance_count(unit: str, model: COUModel) -> int:
```

Comment:
```suggestion
async def get_nova_compute_instance_count(unit: str, model: COUModel) -> int:
```

Reply:
I would prefer keeping the function name short. I improved the docstrings based on your suggestion so I think it is clearer now.

Reply:
Since we are passing the name of unit, it can be `cinder/0` or `Harry Potter`, where both values should fail. So we do not check the name, so I do not see any need to rename this function.

Reply:
So it's two against one :sweat_smile: . No big deal, solving it.